src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/EditIssuePage.kt

```diff
@@ -31,7 +30,11 @@ class EditIssuePage(
 
     fun fillForm(): EditIssuePage {
         summaryLocator.clearAndTypeIfPresent("summary")
-        descriptionLocator.clearAndTypeIfPresent("description")
+        val descriptionFieldId = "description"
+        if(driver.isElementPresent(By.id(descriptionFieldId))){
```

Comment:
Let's reformat, the line is very cramped.
PS. `By.id(descriptionFieldId)` is extractable.